modules/claims_api/app/swagger/claims_api/forms/form_2122_example.json

```diff
@@ -1,59 +1,54 @@
-{ 
-  "data": {
-    "type": "form/21-22",
-    "attributes": {
-      "serviceOrganization": {
-        "poaCode": "1AQ",
-        "firstName": "VSOFirst",
-        "lastName": "VSOLast",
-        "organizationName": "I Help Vets LLC",
-        "address": {
-          "numberAndStreet": "123 NotReal Pkwy",
-          "city": "sometown",
-          "state": "WI",
-          "country": "US",
-          "zipFirstFive": "54703"
-        }
+{
```

Comment:
The diff for this and the other `json` file is deceiving. All I did was strip out the top-level `data` properties, since defining them in the swagger blocks adds them us already

Reply:
Should we be keeping the full payload in the example and remove the extra layer of data form the swagger section that's calling this file? Would be confusing for someone to see the form 2122 example here but it's incorrectly formatted?

Reply:
I don't think it's a problem to leave it out here. this is only used within the swagger code, and the end result is showing what's correct, so I wouldn't consider it misleading.

I'm having a hard time getting the swagger blocks to play nicely if I leave the data property in and getting everything to display properly, so maybe keeping the change but renaming this to `form_2122_example_data.json` or something to be a tad more explicit is a good middle ground?